Routes/account.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express")
+const accountRoutes = express.Router();
+const fs = require('fs');
+
+
+const dataPath = './Details/useraccount.json' 
+
+// util functions 
+
+const saveAccountData = (data) => {
+    const stringifyData = JSON.stringify(data)
+    fs.writeFileSync(dataPath, stringifyData)
+}
+
+const getAccountData = () => {
+    const jsonData = fs.readFileSync(dataPath)
```

Comment:
Import the JSON data. Do not read from file

Reply:
made corrections